### PR TITLE
Update coupled with develop

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
Update the coupled-crow branch with current develop.

### CODE REVIEW

**Comment by WalterKolczynski-NOAA** [comment]
After PR #359 and PR #338.

File: jobs/JGLOBAL_WAVE_PREP [hunk 1]
```diff
@@ -79,13 +79,24 @@ else
       $NLN $DMPDIR/${WAVECUR_DID}.${RPDY} $ROTDIR/${WAVECUR_DID}.${RPDY}
     fi
     export COMIN_WAV_RTOFS=${COMIN_WAV_RTOFS:-$ROTDIR}
-  fi 
+  fi
```
Comment (JessicaMeixner-NOAA) [inline jobs/JGLOBAL_WAVE_PREP:82]: Please see /scratch2/NCEPDEV/climate/Jessica.Meixner/p7update/mergedev/jobs/JGLOBAL_WAVE_PREP and update this file. 

The new section with BRPDY should also be behind the if [ $WW3CURINP = "YES" ] statement and the "COMIN_WAV_ICE" line is repeated.  The COMIN_WAV_RTOFS also needs to be in the if statement block.

**Comment by JessicaMeixner-NOAA** [comment]
@WalterKolczynski-NOAA  I saw that this was "on hold" and based on your comment it was waiting on another PR to be merged, which has now been merged.   There's still the comment that I made before that needs to be addressed, but I wanted to see if I needed to continue my review of this PR or if this was still on hold?

**Comment by WalterKolczynski-NOAA** [comment]
With the move away from coupled-crow, I'm closing this PR but I'll keep the branch for a while just in case.

**Pull request closed without merging**